data/json/items/armor/hoods.json

```diff
@@ -50,6 +50,62 @@
     "proportional": { "weight": 1.25, "volume": 1.3, "price": 1.25 },
     "flags": [ "VARSIZE", "STURDY", "OVERSIZE", "HELMET_COMPAT" ]
   },
+  {
+    "id": "gambeson_hood",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str": "gambeson hood" },
+    "description": "A customized hood made from quilted fabrics. Can be worn under helmets.",
+    "weight": "340 g",
+    "volume": "300 ml",
+    "price": 412,
+    "price_postapoc": 500,
+    "to_hit": -1,
+    "material": [ "cotton" ],
+    "symbol": "[",
+    "looks_like": "balclava",
+    "color": "light_red",
+    "material_thickness": 5,
+    "flags": [ "VARSIZE", "STURDY", "HELMET_COMPAT" ],
+    "armor": [ { "encumbrance": 8, "coverage": 90, "covers": [ "head" ] }, { "encumbrance": 8, "coverage": 45, "cover_vitals": 80, "covers": [ "torso", "arm_l", "arm_r" ],  "specifically_covers": [ "arm_shoulder_l", "arm_shoulder_r", "torso_upper" ], "layers": [ "OUTER" ] } ]
```

Comment:
Keep in mind this covers about 20% of the torso and 10% of the arms overall. I would say it should have 1-2 encumbrance when fitted.